### PR TITLE
gh-89762: Document strftime %G, %V, and %u format specifiers

### PR DESCRIPTION
**Pull request by rruuaanng**
* Issue: gh-89762

----
📚 Documentation preview 📚: https://cpython-previews--124572.org.readthedocs.build/

### CODE REVIEW

**Comment by nineteendo** [comment]
Shouldn't tests be added when this is documented?

**Comment by rruuaanng** [comment]
> Shouldn't tests be added when this is documented?

I don't think testing is needed, because this behavior is already supported in `time.strftime` and there are tests for it.

File: Doc/library/time.rst [hunk 1]
```diff
@@ -515,6 +518,17 @@ Functions
    | ``%Z``    | Time zone name (no characters if no time zone  |       |
    |           | exists). Deprecated. [1]_                      |       |
    +-----------+------------------------------------------------+-------+
+   | ``%G``    | ISO 8601 year (similar to %Y but follows       |       |
+   |           | the rules for the ISO 8601 calendar year).     |       |
+   |           | The ISO year starts with the week that         |       |
+   |           | contains the first Thursday of the calendar    |       |
+   |           | year.                                          |       |
```
Comment (erlend-aasland) [inline Doc/library/time.rst]: 1. Properly mark up `%Y`
2. Remove unneded word.

```suggestion
   | ``%G``    | ISO 8601 year (similar to ``%Y`` but follows   |       |
   |           | the rules for the ISO 8601 calendar year).     |       |
   |           | The year starts with the week that contains    |       |
   |           | the first Thursday of the calendar year.       |       |
```

**Comment by erlend-aasland** [comment]
@rruuaanng: please try to word the PR title as accurately as possible concerning what the PR actually does.

**Review by erlend-aasland — APPROVED** [review]
Thanks for the PR!

I'll let this sit a couple of days to give Paul a chance to chime in.[^1]

[^1]: There is no need to comment further on this remark.

**Comment by erlend-aasland** [comment]
@rruuaanng, why are you pinging me again on this issue?

**Comment by rruuaanng** [comment]
It seems that Paul is not here, so I thought maybe I could ask someone else to review it. I seem to be a bit anxious, please forgive me for this behavior :).

**Comment by erlend-aasland** [comment]
> It seems that Paul is not here, so I thought maybe I could ask someone else to review it. I seem to be a bit anxious, please forgive me for this behavior :).

This is not a urgent issue; please give Paul time to chime in. There is no need to ping Paul nor me.

**Review by pganssle — APPROVED** [review]

**Review by erlend-aasland — APPROVED** [review]

**Pull request merged**